addons/html_editor/static/tests/toolbar.test.js

```diff
@@ -308,11 +308,11 @@ test("toolbar works: can select font size", async () => {
     expect(".o-we-toolbar").toHaveCount(0);
     setContent(el, "<p>[test]</p>");
     await waitFor(".o-we-toolbar");
-    const iframeEl = queryOne(".o-we-toolbar [name='font-size'] iframe");
+    const iframeEl = queryOne(".o-we-toolbar [name='font_size_selector'] iframe");
```

Comment:
It's a bit weird because you added `name="font_size_selector"` and name="font"`, and changed where `font-size` was used, but I don't see a removal of `name="font-size"` anywhere in this commit and I don't see `name="font"` used either. Maybe in the next commits?

EDIT: I guess the `name="font-size"` was automatically generated from the button group that you removed? But then I don't understand why a new `name` was needed rather than simply targeting the name of the new group.

Reply:
as discussed live: it used to be it solo in a group (font-size), and that's no longer the case.